src/app/Mappings/MappingsPage.tsx

```diff
@@ -8,18 +8,13 @@ import {
   Tab,
   TabTitleText,
 } from '@patternfly/react-core';
-import { Mapping, MappingType } from '@app/queries/types';
+import { Mapping, MappingType, MapType } from '@app/queries/types';
 import CreateMappingButton from '@app/Mappings/components/CreateMappingButton';
 import AddEditMappingModal from '@app/Mappings/components/AddEditMappingModal';
 import Mappings from '@app/Mappings/Mappings';
 import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
 
 const MappingsPage: React.FunctionComponent = () => {
-  enum MapType {
-    'Network',
-    'Storage',
-  }
-
   const [activeTabKey, setActiveTabKey] = React.useState<React.ReactText>(0);
```

Comment:
The tabKey can be a string, and we don't need both activeTabKey and activeMapType since they duplicate the same state. See the tabs on ProvidersPage, it's all driven from `activeProviderType` which is just a string.

Reply:
Ah yes, tabKey as string, good call! Updated.